src/Apps/Artist/Routes/Consign/__tests__/ConsignRoute.test.tsx

```diff
@@ -0,0 +1,194 @@
+import { ConsignRouteFixture } from "Apps/__tests__/Fixtures/Artist/Routes/ConsignRouteFixture"
+import { SystemContextProvider } from "Artsy"
+import { renderRelayTree } from "DevTools"
+import React from "react"
+import { graphql } from "relay-runtime"
+import { ConsignRouteFragmentContainer } from "../index"
+
+import { ConsignRoute_Test_QueryRawResponse } from "__generated__/ConsignRoute_Test_Query.graphql"
+
+jest.unmock("react-relay")
+
+jest.mock("Apps/Artist/Routes/Consign/Utils/getConsignmentData", () => ({
+  getConsignmentData: () => {
+    const {
+      artistConsignmentFixture,
+    } = require("Apps/__tests__/Fixtures/Artist/Routes/ConsignRouteFixture")
+
+    return artistConsignmentFixture
+  },
+}))
+
+describe("ConsignRoute", () => {
+  const getWrapper = async (
```

Comment:
This thing breaks my brain, haha. I think what I'm reading here is a mocking of relay? It allows us to get a wrapper in the enzyme sense, right? But it's not really an enzyme wrapper? Sorta confusing and lots of abstraction that I'm trying to understand but I guess in the end if you can use it like the wrappers I'm used to, then the details probably don't matter.

Which I guess makes me wonder if this construct is more of a distraction in this test file than it's worth. Have you considered pulling something like this out into a separate location? Just a thought!

Reply:
This is the typical Relay pattern that we use all over Reaction, and it does return an enzyme wrapper! Lots of goodies in here: https://github.com/artsy/reaction/tree/master/src/DevTools